docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc

```diff
@@ -1425,6 +1425,335 @@ PacketFence switch configuration:
+    20 class NOT_IN_CRITICAL_AUTH do-until-failure
+     10 resume reauthentication
+   event inactivity-timeout match-all
+    10 class always do-until-failure
+     10 clear-session
+   event authentication-success match-all
+    10 class always do-until-failure
+     10 activate service-template DEFAULT_LINKSEC_POLICY_SHOULD_SECURE
+
+for 802.1X Only:
+
+  policy-map type control subscriber DOT1X
+   event session-started match-all
+    10 class always do-until-failure
+     10 authenticate using dot1x priority 10
+   event authentication-failure match-first
+    10 class AAA_SVR_DOWN_UNAUTHD_HOST do-until-failure
+     10 activate service-template CRITICAL_AUTH_VLAN
+     20 activate service-template DEFAULT_CRITICAL_VOICE_TEMPLATE
+     30 activate service-template CRITICAL-ACCESS
+     40 authorize
+     50 pause reauthentication
+    20 class AAA_SVR_DOWN_AUTHD_HOST do-until-failure
+     10 activate service-template CRITICAL_AUTH_VLAN
+     20 activate service-template DEFAULT_CRITICAL_VOICE_TEMPLATE
+     30 activate service-template CRITICAL-ACCES
+     40 pause reauthentication
+     50 authorize
+    30 class DOT1X_FAILED do-until-failure
+     10 terminate dot1x
+    40 class DOT1X_NO_RESP do-until-failure
+     10 terminate dot1x
+    60 class always do-until-failure
+     10 terminate dot1x
+     20 authentication-restart 10800
+   event agent-found match-all
+    10 class always do-until-failure
+     10 authenticate using dot1x priority 10
+   event aaa-available match-all
+    10 class IN_CRITICAL_AUTH do-until-failure
+     10 clear-session
+    20 class NOT_IN_CRITICAL_AUTH do-until-failure
+     10 resume reauthentication
+   event inactivity-timeout match-all
+    10 class always do-until-failure
+     10 clear-session
+   event authentication-success match-all
+    10 class always do-until-failure
+     10 activate service-template DEFAULT_LINKSEC_POLICY_SHOULD_SECURE
+
+Interface Template (802.1x Mac Authentication):
+
+  template identity-template-mab
+   dot1x pae authenticator
+   spanning-tree portfast edge
+   switchport access vlan 1
+   switchport mode access
+   switchport voice vlan 100
+   mab
+   access-session host-mode multi-domain
+   access-session control-direction in
+   access-session closed
+   access-session port-control auto
+   authentication periodic
+   authentication timer reauthenticate server
+   service-policy type control subscriber DOT1X_MAB
+
+Interface Template (Mac Authentication):
+
+  template identity-template-macauth
+   dot1x pae authenticator
+   spanning-tree portfast edge
+   switchport access vlan 1
+   switchport mode access
+   switchport voice vlan 100
+   mab
+   access-session host-mode single-host
+   access-session control-direction in
+   access-session closed
+   access-session port-control auto
+   authentication periodic
+   authentication timer reauthenticate server
+   service-policy type control subscriber MACAUTH
+
+Interface Template (801.1x):
```

Comment:
Wow 801.1x thats outdated, I heard 803.1x is coming out next week.

We need to keep up